Update slurm tests returned perf_values

jgphpc opened the pull request:
- [x] follow-up of #375
- [x] ~blocked by~ https://github.com/eth-cscs/cscs-reframe-tests/pull/402
- [x] Run with:
```console
UENV=prgenv-gnu/25.06:rc5 ./R -c cscs-reframe-tests.git/checks/system/slurm/slurm.py -J p=normal -r
```

- You can manually trigger 1 (or more) CI pipelines by adding a `cscs-ci run alps-daint-uenv` comment in the Pull Request
- By default all UENVs will be tested, however you can restrict to a single UENV with: `cscs-ci run alps-daint-uenv;MY_UENV=cp2k/2025.1:v2`

gppezzi commented:
@jgphpc the tests are passing on Daint, but are you addressing the gres issue reported on SD-66684?
 
It is unclear, as the old test (which you are deleting in this PR) still fails and the updated one (DefaultRequestGPUSetsGRES) doesn't.

jgphpc commented:
> @jgphpc the tests are passing on Daint, but are you addressing the gres issue reported on SD-66684?
> 
> It is unclear, as the old test (which you are deleting in this PR) still fails and the updated one (DefaultRequestGPUSetsGRES) doesn't.

- I did not delete gres_gpu.py, I moved it into slurm.py (no reason to keep it separate). 
- This PR is not really addressing the issue in SD-66684, it's more about sending the right data to Elastic (I updated the tests. in the file while I was reading it).

jgphpc commented:
https://reframetalk.slack.com/archives/CF8ENLUDB/p1756210315486089
